turbo.json

```
@@ -13,26 +10,62 @@
       ],
       "outputLogs": "new-only",
       "outputs": [
-        "src/artifacts/**",
-        "src/gen/**",
-        "gen/**"
+        "src/artifacts/**"
+       ]
+    },
+    "compact:fast": {
+      "env": [
+        "COMPACT_HOME",
+        "SKIP_ZK"
+      ],
+      "inputs": [
+        "src/**/*.compact"
+      ],
+      "outputLogs": "new-only",
+      "outputs": [
+        "src/artifacts/**"
       ]
     },
     "test": {
-      "outputs": [],
+      "dependsOn": [
+        "compact:fast",
+        "^compact:fast"
+      ],
+      "env": [
+        "COMPACT_HOME"
+      ],
       "cache": false
     },
     "build": {
       "dependsOn": [
-        "^build",
-        "compact"
+        "^build"
       ],
       "env": [
         "COMPACT_HOME"
       ],
       "inputs": [
         "src/**/*.ts",
-        "src/**/*.mts",
+        "src/**/*.tsx",
+        "!src/**/*.test.ts",
+        "!tests/**/*.ts",
+        "tsconfig.json",
+        "tsconfig.build.json",
+        ".env"
+      ],
+      "outputs": [
+        "dist/**"
+      ]
```

```suggestion
      "outputs": [
        "dist/**",
        ".next/**"
      ]
```
For the app output, right?